src/cudamatrix/cu-kernels.cu

```diff
@@ -1763,6 +1763,31 @@ static void _group_max(Real *y, const Real *x, MatrixDim d, int src_stride,
   }
 }
 
+template<typename Real>
+__global__
+static void _parametric_relu(Real*y, const Real*x, MatrixDim d, int src_stride, const Real*a, const Real*b) {
+  int i = blockIdx.x * blockDim.x + threadIdx.x;
+  int j = blockIdx.y * blockDim.y + threadIdx.y;
+  int dst_index = i + j*d.stride, src_index = i + j * src_stride;
+  if(i < d.cols && j < d.rows) {
```

Comment:
There should be space between 'if' and '(', it is part of the Google style. The script which checks the style is here: https://github.com/kaldi-asr/kaldi/blob/master/src/doc/cpplint.py